docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   app:
-    image: decidim/decidim:0.24.3
+    image: decidim/decidim:0.25.2
```

Comment:
Change it to 0.26.4 as it's the latest image available in Dockerhub for 0.26.x https://hub.docker.com/r/decidim/decidim/tags?page=1&name=0.26

Reply:
Change the version in Dockerfile too